src/test/java/reposense/git/GitLsTreeTest.java

```diff
@@ -85,7 +84,7 @@ public void unix_cloneInvalidWindowsFilePaths_success() {
      * @throws InvalidFilePathException if the repository contains invalid file paths that are not compatible with
      * Windows.
      */
-    private void validateFilePaths(RepoConfiguration config) throws IOException, InvalidFilePathException {
+    private void validateFilePaths(RepoConfiguration config) throws Exception {
```

Comment:
For this utility method, let's keep the original `throws` clause as well?  (i.e. specify IOException and InvalidFilePathException in the clause)

Reply:
@anubh-v are you referring to the javadocs in your comment?

I'm thinking of whether to include the line
```java
@throws Exception if it fails to create/delete the folders, or if the repository contains invalid file paths .....
```

in the javadocs

Reply:
My apologies - I missed this message.
I was referring to the `throws` clause of the method, not its the javadoc.
For this method,`validateFilePaths`, I think both the `throws` clause and the javadoc should not be modified.

This method is a utility method used for writing tests. Hence, it would be good to have the
specific exceptions listed in `throws` clause, so developers who write tests can quickly understand its behaviour.

@yash-chowdhary has reversed his changes, so the method is OK now